packages/connection/src/class-rest-connector.php

```diff
@@ -251,42 +245,21 @@ public static function get_user_permissions_error_msg() {
 	 *
 	 * @since 8.8.0
 	 *
-	 * @param WP_REST_Request $request The request sent to the WP REST API.
-	 *
 	 * @return \WP_REST_Response|WP_Error
 	 */
-	public function connection_reconnect( WP_REST_Request $request ) {
-		$params = $request->get_json_params();
-
+	public function connection_reconnect() {
 		$response = array();
 
 		$next = null;
 
-		switch ( $params['action'] ) {
-			case 'reconnect':
-				$result = $this->connection->restore();
-
-				if ( is_wp_error( $result ) ) {
-					$response = $result;
-				} elseif ( is_string( $result ) ) {
-					$next = $result;
-				} else {
-					$next = true === $result ? 'completed' : 'failed';
-				}
+		$result = $this->connection->restore();
 
-				break;
-			case 'reconnect_force':
```

Comment:
@sergeymitr I found no current usage of `reconnect_force` action so I cleaned this as up well. Please let me know if I shouldn't in order to restore it. Thanks!

Reply:
Hi @fgiannar.
The changes look good, but I think we are going to need the `reconnect_force` at some point.
We are planning to add the "Refresh Connection" button into the Connection Manager (`p9dueE-1vM-p2`), and `reconnect_force` is the action we would perform.

Do you think we should clean it up now and bring it back later, or it makes sense to leave it in the code not as a technical debt, but as an API for not-yet-implemented UI?

Reply:
I guess it depends on the implementation.
Now we require an action. If we want to keep this logic then it's better to revert most of the changes introduced here.
If we prefer to use an optional `force` parameter that would trigger the full reconnection, then I guess it would make sense to clean it up now and bring it back later.
Let me know your thoughts!

Reply:
I like the idea of replacing it with the `force` parameter, we don't really need to support the `actions` in there.